packages/donor/src/screens/UpcommingDonation/EventCalendarLink.tsx

```diff
@@ -62,17 +53,37 @@ export default function EventCalendarLink({
     window.open(googleEventUrlWithParams, "_blank");
   };
 
+  const addToIOSCalendar = () => {
+    const calendarUrl = [
+      "BEGIN:VCALENDAR",
+      "VERSION:2.0",
+      "BEGIN:VEVENT",
+      "URL:" + document.URL,
+      "DTSTART:" + CleanIsoFormatDate(eventDateStart),
+      "DTEND:" + CleanIsoFormatDate(eventDateEnd),
+      "SUMMARY:תרומת טרומבוציטים",
+      "DESCRIPTION:" + eventDescription,
+      "LOCATION:" + eventLocation,
+      "END:VEVENT",
+      "END:VCALENDAR",
+    ].join("\n");
+
+    const blob = new Blob([calendarUrl], {
+      type: "text/calendar;charset=utf-8",
+    });
+    const ret = window.URL.createObjectURL(blob);
+    window.open(ret, "_blank");
+  };
+
   return (
     <div>
       {DeviceUtils.getMobileOperatingSystem() === "android" ? (
         <div className={styles.link} onClick={addToGoogleCalendar}>
           הוספה ליומן
         </div>
       ) : (
-        <div className={styles.link}>
-          <ICalendarLink event={getAppleCalendarEvent()}>
-            הוספה ליומן
-          </ICalendarLink>
+        <div className={styles.link} onClick={addToIOSCalendar}>
+          הוספה ליומן
```

Comment:
These 2 divs are the same, they only have different onClicks.

I suggest you merge them, and create another onClick function that will call the relevant `addTo...` based on the device